jobs/build_docker/build_docker.sh

```diff
@@ -53,38 +53,44 @@ packagerefs-commit
 
```

Comment:
I think the docker build step of on-web-ui should only build docker for itself. the same as ucs/smi...etc.
based on this definition,  most part of this build_docker.sh is not relevant to on-image-builder , but to build rackhd..
I suggest write a new file inside on-web-ui:
```

#download manifest, which only contains on-web-ui/on-wss
curl --user $BINTRAY_CREDS -L "$MANIFEST_FILE_URL" -o rackhd-manifest

#clone
./build-config/build-release-tools/HWIMO-BUILD ./build-config/build-release-tools/application/reprove.py \
--manifest rackhd-manifest \
--builddir ./$CLONE_DIR \
--jobs 8 \
--force \
checkout \
packagerefs-commit

#rsync -r ./$CLONE_DIR/RackHD/ ./build/ # can be deleted ?

REPOS=$(ls $CLONE_DIR )
# Assume the on-build-config repo has been cloned to build-config folder
pushd build-config/build-release-tools/
./docker_build.sh $WORKSPACE/$CLONE_DIR $IS_OFFICIAL_RELEASE "$REPOS"
#cp $WORKSPACE/$CLONE_DIR/build_record $WORKSPACE
cp $WORKSPACE/$CLONE_DIR/build_record . # copy build_record to current directory for stash
popd

# save docker image to tar
image_list=`cat $WORKSPACE/$CLONE_DIR/build_record | xargs`
docker save -o rackhd_docker_images.tar $image_list

```